### PR TITLE
Docs/refactor

### PR DESCRIPTION
**Pull request by dehann**
Major refactor

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -6,15 +6,26 @@ makedocs(
...
-        "Functions" => "func_ref.md"
+        "Getting Started" => [
+            "Installation" => "installation_environment.md"
+        ],
+        "Concepts" => [
+            "Caesar Concepts" => "concepts/concepts.md",
+            "Building Factor Graphs" => "concepts/building_graphs.md",
+            "Adding New Variables and Factors" => "concepts/adding_variables_factors.md",
+            "Arena Visualization" => "concepts/arena_visualizations.md",
+            "Using Caesar's Multi-Language Support" => "concepts/zmq.md",
```
Comment (pvazteixeira) [inline docs/make.jl]: maybe `multilang` instead of `zmq` wherever possible?

Comment (dehann) [reply docs/make.jl]: +1

multilang (using ZMQ)

Comment (GearsAD) [reply docs/make.jl]: Done! Thanks :gear:

**Comment by dehann** [comment]
partially fixes #72

**Comment by dehann** [comment]
helps resolve #136

**Pull request merged**